static/js/settings_account.js

```diff
@@ -383,6 +394,12 @@ export function set_up() {
     function clear_password_change() {
         // Clear the password boxes so that passwords don't linger in the DOM
         // for an XSS attacker to find.
+        $("#old_password").attr("type", "password");
+        $("#new_password").attr("type", "password");
+        const password_toggle_one = $("#old_password ~ .password_visibility_toggle");
+        const password_toggle_two = $("#new_password ~ .password_visibility_toggle");
```

Comment:
These would be better named as E.g. `old_password_visibility_toggle`.  Also would it be better for this to call a helper exported from `common.js` twice rather than copying parts of the implementation here?